src/styles/blocks/header.scss

```diff
@@ -0,0 +1,16 @@
+@import "../utils/variables";
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__logo {
```

Comment:
For logo can try display flex,  justify-content and align-items, because logo is not in the center